<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生选课系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="ShowView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>601</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="add_student"/>
    <addaction name="add_course"/>
    <addaction name="show_course"/>
    <addaction name="show_studnet"/>
    <addaction name="delete_student"/>
    <addaction name="course_student"/>
    <addaction name="delete_course"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="FileSave"/>
    <addaction name="FileRead"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="add_student"/>
   <addaction name="add_course"/>
   <addaction name="delete_course"/>
   <addaction name="show_course"/>
   <addaction name="show_studnet"/>
   <addaction name="course_student"/>
  </widget>
  <action name="add_student">
   <property name="text">
    <string>添加学生</string>
   </property>
  </action>
  <action name="add_course">
   <property name="text">
    <string>添加课程</string>
   </property>
  </action>
  <action name="show_course">
   <property name="text">
    <string>显示学生课表</string>
   </property>
  </action>
  <action name="show_studnet">
   <property name="text">
    <string>显示学生信息</string>
   </property>
  </action>
  <action name="FileSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="FileRead">
   <property name="text">
    <string>写入</string>
   </property>
  </action>
  <action name="delete_student">
   <property name="text">
    <string>删除学生</string>
   </property>
  </action>
  <action name="course_student">
   <property name="text">
    <string>特定课程学生</string>
   </property>
  </action>
  <action name="delete_course">
   <property name="text">
    <string>删除课程</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>add_student</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_course</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
